go.mod

```
@@ -0,0 +1,3 @@
+module github.com/git-lfs/go-netrc
+
+go 1.20
```

I don't think we want to force users to use Go 1.20.  We'd probably prefer to specify the oldest version that works here, which is going to be something more like 1.13.

---

I don't have a strong preference here

```suggestion
go 1.13
```

I did due-diligence and check to see if it work with 1.13 though

```bash
> podman run --rm -it golang:1.13 bash

root@f78a91ddd659:/go# git clone https://github.com/git-lfs/go-netrc/

root@f78a91ddd659:/go# cd go-netrc/

root@f78a91ddd659:/go/go-netrc# go test ./...
ok      _/go/go-netrc/netrc     0.002s

root@f78a91ddd659:/go/go-netrc# go build ./...

root@f78a91ddd659:/go/go-netrc# go mod init github.com/git-lfs/go-netrc
root@f78a91ddd659:/go/go-netrc# cat go.mod
module github.com/git-lfs/go-netrc

go 1.13
```

so everything works.